praw/models/reddit/widgets.py

```diff
@@ -1002,7 +1002,7 @@ def upload_image(self, file_path):
 class Widget(PRAWBase):
     """Base class to represent a Widget."""
 
-    @property
+    @cachedproperty
```

Comment:
I can't comment on the part below for some reason, but the whole
```python
if thing is None
    thing = Something()
return thing
```
in the body of the method should just become
```python
return Something()
```
That's the beauty of `@cachedproperty`.

Reply:
I assumed it was a simple find/replace, so I'll make the change.